src/layouts/sidebar-sidecar/index.tsx

```diff
@@ -68,12 +68,11 @@ const SidebarSidecarLayoutContent = ({
 		!isDesktop && sidebarIsVisible
 	)
 
-	const SidebarContent = (): ReactElement => {
-		if (AlternateSidebar && !sidebarProps?.menuItems) {
-			return <AlternateSidebar {...sidebarProps} />
-		}
-
-		return <Sidebar {...sidebarProps} />
+	let sidebarContent = null
+	if (AlternateSidebar && !sidebarProps?.menuItems) {
+		sidebarContent = <AlternateSidebar {...sidebarProps} />
+	} else {
+		sidebarContent = <Sidebar {...sidebarProps} />
```

Comment:
I refactored this to not create a component in the component body, as it was causing the sidebar to fully re-mount on each render of the layout (ultimately resets component state)